### DISCUSSION TITLE
我不知道是我的操作有问题还是nx本身就是这样

### DISCUSSION THREAD

**Discussion by flycran** [nrwl/nx#26300] [General]
我刚开始学习nx，遇到一些问题，让我对nx的好感很差，我不知道是我的操作有问题还是nx本身就是这样。

- 首先一旦我安装或卸载依赖或者是其他操作，都会有一个命令行窗口一闪而过，它很影响我的心情。
- 我想添加一些路径别名，于是我在`tsconfig`添加`"paths": { "@/*": ["./src/*"] }`，但它不起作用，甚至让`tsconfig.base`的`paths`一并失效了。
- 项目的文件夹经常会被占用导致无法删除，即使是使用`nx g remove`也无济于事。

这些痛点让我丧失了学习nx的动力

**Comment by jaysoo** [comment]
Hi @flycran, thanks for taking the time to open this discussion.

1. There is no way to extend `compilerOptions.paths` in tsconfig. If you set it in your app, then it will override the `tsconfig.base.json` file. The TS team has [a proposal to merge options](https://github.com/microsoft/TypeScript/issues/57486) but there is no guarantee or ETA on it. That said, we are looking to improve our TSC setup so stay tuned.
2. If you are having issues with `nx g remove` please report an issue with a reproduction and we can investigate.
